apis/Google.Cloud.Firestore/Google.Cloud.Firestore/Query.cs

```diff
@@ -78,17 +79,17 @@ private protected Query(FirestoreDb database, DocumentReference parent, string c
         private Query(
             QueryRoot root,
             int offset, (int count, LimitType type)? limit,
-            IReadOnlyList<InternalOrdering> orderings, IReadOnlyList<InternalFilter> filters, IReadOnlyList<FieldPath> projections,
-            Cursor startAt, Cursor endAt)
+            IReadOnlyList<InternalOrdering> orderings, IReadOnlyList<FieldPath> projections,
```

Comment:
If you just replace the existing parameter by changing the name and type, there are far fewer changes elsewhere. (You're not logically changing what's needed for construction after all - just how it's expressed.)

Indeed, it might end up being cleanest to do this in two commits: one which changes from `IReadOnlyList<InternalFilter> filters` to `Filter.CompositeFilter filters`, and then another commit which *just* changes the name later. That way all the "churn" changes are isolated.